sources/packages/backend/workflow/test/2022-2023/assessment-gateway/assessment-gateway.originalAssessment.e2e-spec.ts

```diff
@@ -41,14 +42,31 @@ describe(`E2E Test Workflow assessment gateway on original assessment for ${PROG
     // Arrange
 
     // Assessment consolidated mocked data.
-    const assessmentConsolidatedData: AssessmentConsolidatedData = {
-      ...createFakeConsolidatedFulltimeData(PROGRAM_YEAR),
+    const assessmentDataOnSubmit: AssessmentConsolidatedData = {
       assessmentTriggerType: AssessmentTriggerType.OriginalAssessment,
+      ...createFakeConsolidatedFulltimeData(PROGRAM_YEAR),
       ...createFakeSingleIndependentStudentData(),
       // Application with PIR not required.
       studentDataSelectedOffering: 1,
     };
 
+    const workersMockedData = createWorkersMockedData([
+      createLoadAssessmentDataTaskMock({
+        assessmentConsolidatedData: assessmentDataOnSubmit,
+        subprocess:
+          WorkflowSubprocesses.LoadConsolidatedDataSubmitOrReassessment,
+      }),
+      createLoadAssessmentDataTaskMock({
+        assessmentConsolidatedData: assessmentDataOnSubmit,
```

Comment:
Submit and PreAssessment sub process are using the same data?

Reply:
I meant can PreAssessment can't have any mocked data at all?

Reply:
We can now. I changed it as below.

It was like this

![image](https://user-images.githubusercontent.com/61259237/230968828-2d422bb8-194c-4584-8c3f-0d7282fdfe2b.png)

And now it is like below

![image](https://user-images.githubusercontent.com/61259237/230968875-072b7660-00c3-4d58-8614-ffda14e670d5.png)